README.rst

```diff
@@ -446,3 +446,19 @@ After initialization, you are ready to retrieve data
 
     except NotFoundError:
         # File does not exist
+
+Accessing Thoth Data on the Operate-First Public Bucket
+=======================================================
+
+A public instance of Thoth's database is available on the `Operate-First Public Bucket
+<https://github.com/operate-first/apps/blob/4008fc6117314a6b689f4ffa9264d423db57a99f/docs/content/odh/trino/access_public_bucket.md>`__ for external contributors to start developing components of Thoth.
```

Comment:
should this reference the master branch rather than a specific commit?

Reply:
Right, thank you!

Reply:
It might be better to reference instructions in the `datasets` repository to maintain instructions only in one place.

Reply:
Good idea, I will reference a link to the corresponding section in `datasets` once https://github.com/thoth-station/datasets/pull/67/files is merged :+1:

Reply:
Changed by https://github.com/thoth-station/storages/pull/2565/commits/f271abc82697b18fdf1ddd1cfed1660fa1f76c86